cephmetrics-pull-requests/config/definitions/cephmetrics-pull-requests.yml

```diff
@@ -0,0 +1,54 @@
+- job:
+    name: cephmetrics-pull-requests
+    project-type: freestyle
+    defaults: global
+    concurrent: true
+    display-name: 'Cephmetrics: Pull Requests'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - github:
+          url: https://github.com/ceph/cephmetrics/
+    logrotate:
+      daysToKeep: 15
+      numToKeep: 30
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+
+    parameters:
+      - string:
+          name: sha1
+          description: "A pull request ID, like 'origin/pr/72/head'"
+
+    triggers:
+      - github-pull-request:
+          admin-list:
+            - zmc
+            - pcuzner
+            - b-ranto
+            - GregMeno
+          org-list:
+            - ceph
+          only-trigger-phrase: false
+          github-hooks: true
+          permit-all: false
```

Comment:
Oh, good to know. I'm not actually sure which we want here, but I suppose we could leave this as-is until we need to change it (if we ever do)